schunk_sdh/msg/TemperatureArray.msg

```diff
@@ -0,0 +1,2 @@
+Header header
+Temperature[] temperature_list
```

Comment:
I found having a list like `{ {name1, value1}, {name2, value2}, ... }` (e.g. a list of key-value pairs like `std::map`) makes iterating the list and searching for specific `name`s easier. In a list like `{ {name1, name2, ...}, {value1, value2, ...} }`, you would first need to find the index corresponding to a `name` to access its `value` from the other list.

But if you prefer a structure like `JointState`, I will change the message type.